### PR TITLE
Add KogeFarm Kava TVL

### PR DESCRIPTION
**Pull request by kogecoindev**
This commit adds KogeFarm's Kava TVL.

The script uses the **unwrapUniswapLPs** and **getSinglePositions** helper functions to obtain token balances, and getFixBalances to get the price for **wKava**. 

A slight under-estimation occurs because some Kava specific assets (ie. Ruby and RShare) are not priced by CoinGecko. The difference is minor, however.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/kogefarm exports TVL: 
        
 
 
    polygon-staking           92.53 k
    kava                      6.41 k
    polygon-pool2             26.12 k
    moonriver                 30.27 k
    polygon                   2.41 M
    fantom                    22.35 k
    staking                   92.53 k
    pool2                     26.12 k
    
    total                    2.47 M

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "blakejs": "^1.2.1",
     "borsh": "^0.7.0",
     "curve25519-js": "^0.0.4",
+    "defillama-adapters": "^0.0.1-security.9",
```
Comment (waynebruce0x) [inline package.json:28]: 'This package contained malicious code and was removed from the registry by the npm security team. A placeholder was published to ensure users are not affected in the future.'

**Comment by llamatester** [comment]
The adapter at projects/kogefarm exports TVL: 
        
 
 
    polygon-staking           86.91 k
    kava                      5.77 k
    moonriver                 29.33 k
    polygon-pool2             24.50 k
    fantom                    21.82 k
    polygon                   2.34 M
    staking                   86.91 k
    pool2                     24.50 k
    
    total                    2.39 M

**Comment by kogecoindev** [comment]
Hi! We have removed the changes to package.json and package-lock.json in response to feedback, so all changes are local to the KogeFarm folder now.

**Pull request merged**